goodRuns/v1/GoldenRunList_ConductorFile.sh

```
@@ -0,0 +1,1336 @@
+}
+
+# If 'dontGenerateFileLists' is not set => run CreateDstList.pl on the final .list
+# to build run-specific .list files. If the .list is missing => skip.
+generate_dst_lists() {
+    if $DONT_GENERATE_FILELISTS; then
+        echo "[INFO] Skipping DST list generation due to 'dontGenerateFileLists'."
+        return
+    fi
+
+    echo "Generating DST lists for the main scenario..."
+    cd "${workplace}/../dst_list"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)